app/models/chart_data.rb

```diff
@@ -56,6 +56,12 @@ def customised_chart_config(chart_manager)
     if @chart_config_overrides[:mpan_mprn].present?
       chart_config[:meter_definition] = @chart_config_overrides[:mpan_mprn].to_i
     end
+    if @chart_config_overrides[:series_breakdown].present?
+      chart_config[:series_breakdown] = @chart_config_overrides[:series_breakdown].to_sym
+    end
+    if @chart_config_overrides[:date_ranges].present?
+      chart_config[:timescale] = @chart_config_overrides[:date_ranges].map {|range| { daterange: range }}
+    end
```

Comment:
Maybe not with these changes - but I wonder whether at some point we should refactor Chart Data and separate out the config management in to something else which is possibly a bit more testable in isolation

Reply:
Yes, there's an obvious split in this file. Hard to test the config stuff when you have calls to the drilldown happening too.

Reply:
I've done a first-pass at moving this out (to get the coverage up a bit!).